src/Stripe.net/Services/_interfaces/INestedCreatable.cs

```diff
@@ -0,0 +1,14 @@
+namespace Stripe
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface INestedCreatable<T, O>
+        where T : StripeEntity
+        where O : BaseOptions
+    {
+        T Create(string parentId, O createOptions, RequestOptions requestOptions = null);
+
+        Task<T> CreateAsync(string parentId, O createOptions, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Reading this makes me think it should have failed for UsageRecordService after all. Is there a bug somewhere that does not raise an error if the interface does not have a defined method?